src/Crypto/RNCryptor/Types.hs

```diff
@@ -99,8 +113,9 @@ renderRNCryptorHeader RNCryptorHeader{..} =
 -- A convenient datatype to avoid carrying around the AES cypher,
 -- the encrypted key and so on and so forth.
 data RNCryptorContext = RNCryptorContext {
-        ctxHeader :: RNCryptorHeader
-      , ctxCipher :: AES
+        ctxHeader  :: RNCryptorHeader
+      , ctxCipher  :: AES128
```

Comment:
I assume this is being passed to cryptonite. The correct setting there should be `AES256` according to their docs. Don't be confused when you see "AES128" show up in the Swift code. The "128" there refers to block size, not key size. But in cryptonite, the 128/256 refers to key size.